### ISSUE TITLE
nfts not showing on opensea testnet rinkeby erc721@4.1.0

### ISSUE THREAD

**Issue by muneebsoftblock** [chiru-labs/ERC721A#350]
opensea https://testnets.opensea.io/assets/rinkeby/0x2301334d923434Ff569ba4fD7FC19C35758086bE/1

rinkeby solidity code https://rinkeby.etherscan.io/address/0x2301334d923434ff569ba4fd7fc19c35758086be#code

nfts were showing in version 3.3.0 erc721a but not on erc721a 4.1.0

**Comment by Vectorized** [comment]
Looks like it’s an issue with OpenSea’s Transfer event listener. Did some tests recently and seems like they are sometimes unable to detect the first token minted. 

Try minting more tokens or transferring the first token after minting.

**Comment by muneebsoftblock** [comment]
okay Thanks I got it. Thanks for explanation. You are genius.

**Comment by muneebsoftblock** [comment]
Issue not fixed

**Comment by halversonmd** [comment]
I had this same issue. Please see my solution [here](https://github.com/chiru-labs/ERC721A/issues/354#issuecomment-1181978531).

**Comment by intenex** [comment]
yeah - this is a supportsInterface issue; see the migration notes here on how to fix it: https://chiru-labs.github.io/ERC721A/#/migration?id=supportsinterface

Specifically, update your supportsInterface to be:

```
function supportsInterface(
    bytes4 interfaceId
) public view virtual override(ERC721A, ERC2981) returns (bool) {
    // Supports the following `interfaceId`s:
    // - IERC165: 0x01ffc9a7
    // - IERC721: 0x80ac58cd
    // - IERC721Metadata: 0x5b5e139f
    // - IERC2981: 0x2a55205a
    return 
        ERC721A.supportsInterface(interfaceId) || 
        ERC2981.supportsInterface(interfaceId);
}
```

**Comment by Xuantho78** [comment]
![Uploading 8B07BEE7-FF96-4947-97FD-ECE9586C073B.jpeg…]()

**Comment by Xuantho78** [comment]
Ngtho301178@gmail.com
